lib/pxcontext.js

```diff
@@ -11,7 +11,7 @@ class PxContext {
         const mobileSdkHeader = 'x-px-authorization';
         const mobileSdkOriginalTokenHeader = 'x-px-original-token';
         const vidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/;
-
+        this.startEnforcerTime = Date.now();
```

Comment:
Isn't there a lot of other stuff that happens prior to the context being created? I would imagine (based on the name) that we want a timestamp as soon as the `enforce()` function is called.

Reply:
 I wanted to define a variable that will be saved In the enforcer context.
I did that during the initialization context phase to make it clear.
I found there is no difference after comparing this point to the point where the enforce() method is called.

Reply:
Okay, if there's no difference then that's fine